services/workflows-service/src/workflow/dtos/create-collection-flow-url.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsNumber, IsOptional } from 'class-validator';
+
+export class CreateCollectionFlowUrlDto {
+  @ApiProperty({
+    required: true,
+  })
+  @IsNotEmpty()
```

Comment:
Isn't `@IsString()` missing? If not, adding it just for clarity sake might be a good idea.